JUnit version 4.11
.[FINE] Launching new model instance modbat.test.ChooseTest-1...
[FINE] choose(0, 10) = 0
[FINE] choose(0, 10) = 0
E

There was 1 failure:
1) test782de49f112cb820(modbat.test.TestChooseTest)
java.lang.AssertionError: assertion failed
	at scala.Predef$.assert(Predef.scala)
	at modbat.dsl.Model$.assert(Model.scala)
	at modbat.dsl.Model$class.assert(Model.scala)
	at modbat.test.ChooseTest.assert(ChooseTest.scala)
	at modbat.test.ChooseTest$$anonfun$3.apply$mcV$sp(ChooseTest.scala)
	at modbat.test.ChooseTest$$anonfun$3.apply(ChooseTest.scala)
	at modbat.test.ChooseTest$$anonfun$3.apply(ChooseTest.scala)
	at modbat.mbt.MBT.executeTransition(MBT.scala)
	at modbat.test.TestChooseTest.test782de49f112cb820(TestChooseTest.java)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java)
	at java.lang.reflect.Method.invoke(Method.java)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java)
	at org.junit.runners.ParentRunner.run(ParentRunner.java)
	at org.junit.runners.Suite.runChild(Suite.java)
	at org.junit.runners.Suite.runChild(Suite.java)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java)
	at org.junit.runners.ParentRunner.run(ParentRunner.java)
	at org.junit.runner.JUnitCore.run(JUnitCore.java)
	at org.junit.runner.JUnitCore.run(JUnitCore.java)
	at org.junit.runner.JUnitCore.run(JUnitCore.java)
	at org.junit.runner.JUnitCore.runMain(JUnitCore.java)
	at org.junit.runner.JUnitCore.runMainAndExit(JUnitCore.java)
	at org.junit.runner.JUnitCore.main(JUnitCore.java)

FAILURES!!!
Tests run: 1,  Failures: 1

